.github/workflows/documentation.yml

```
@@ -38,6 +38,7 @@ jobs:
       - name: "Add orphan metadata where needed"
         run: |
           printf '%s\n\n%s\n' ":orphan:" "$(cat docs/en/sidebar.rst)" > docs/en/sidebar.rst
+          printf '%s\n\n%s\n' ":orphan:" "$(cat docs/en/reference/attributes-reference.rst)" > docs/en/reference/attributes-reference.rst
```

Is `:orphan:` actually required given this is referenced in `sidebar.rst` and `validation-of-documents.rst`?

---

Oups, that's the wrong file.
```suggestion
          printf '%s\n\n%s\n' ":orphan:" "$(cat docs/en/reference/attributes-reference.rst)" > docs/en/reference/annotations-reference.rst
```